console/console-init/ui/src/components/TypeAheadSelect/TypeAheadSelect.tsx

```diff
@@ -55,7 +55,7 @@ const TypeAheadSelect: React.FunctionComponent<ITypeAheadSelectProps> = ({
           value={TypeAheadMessage.MORE_CHAR_REQUIRED}
           isDisabled={true}
           key="2"
-        />
+        />,
```

Comment:
Is there any specific reason for adding comma?

Reply:
This has automatically added because of prettier.